s3/Dockerfile

```diff
@@ -19,9 +19,12 @@ RUN rpm --import https://packages.confluent.io/rpm/5.2/archive.key && yum makeca
 yum install -y epel-release && \
 rpm -ivh https://download.ceph.com/rpm-luminous/el7/noarch/ceph-release-1-1.el7.noarch.rpm && \
 yum --enablerepo=epel-testing install -y lttng-ust libradosstriper-devel librados2-devel librdkafka-devel && \
+yum -y install mysql && \
 yum clean all && \
 rm -rf /var/cache/yum
 
```

Comment:
Could you please remove those extra blank lines?

Reply:
done